developers/uploader/uploader-js.md

```diff
@@ -0,0 +1,48 @@
+# Uploader.js
+
+<a href="https://github.com/oceanprotocol/uploader.js" target="_blank">Uploader.js</a> is a robust TypeScript library that serves as a vital bridge to interact with the Ocean Uploader API. It simplifies the process of managing file storage uploads, obtaining quotes, and more within the Ocean Protocol ecosystem. This library offers developers a straightforward and efficient way to access the full range of Uploader API endpoints, facilitating seamless integration of decentralized storage capabilities into their applications. 
+
+Whether you're building a decentralized marketplace, a content management system, or any application that involves handling digital assets, Uploader.js provides a powerful toolset to streamline your development process and enhance your users' experience.
+
+### Browser Usage
+
+Ensure that the Signer object (signer in this case) you're passing to the function when you call it from the browser is properly initialized and is compatible with the browser. For instance, if you're using something like MetaMask as your Ethereum provider in the browser, you'd typically use the ethers.Web3Provider to generate a signer.
+
+1. Setting up a Signer: with MetaMask or similar browser wallets, you can set up an ethers signer as follows:
+
+```
```

Comment:
please add the code type here is it's properly highlighted

Reply:
I'm a bit confused here. What do you mean exacly? line 13 starts as a code block for the code in line 14-15. It's already a code block. 

Or you mean something what's in line 11?

Reply:
When we have a block of code is important to specify the type of code it contains as well so it's properly highlighted

Reply:
Example: ```bash, ```json or ```typescript

Reply:
got it, thanks 👍🏽